conf/app.conf.example

```diff
@@ -3,7 +3,7 @@ httpport = 8000
 runmode = dev
 copyrequestbody = true
 driverName = mysql
-dataSourceName = root:root@tcp(localhost:3306)/
+dataSourceName = root:password@tcp(localhost:3306)/
```

Comment:
why?

Reply:
dsn format was not quite clear to me. I had a hard time understanding what's what to build the project locally.

I believe this file is only here to provide example and my version provides better example.